pom.xml

```
@@ -15,7 +15,7 @@
 
     <artifactId>nakadi-producer-reactor</artifactId>
     <groupId>org.zalando</groupId>
-    <version>20.3.1</version>
+    <version>20.3.2</version>
```

I think this counts as more than a patch version, given that it introduces a new feature.
```suggestion
    <version>20.4.0</version>
```
(This would also have to be changed in the other places.)